### ISSUE TITLE
BUG in page numbering, when converting docx to pdf using the API

### ISSUE THREAD

**Issue by reinaldoteixeira** [ONLYOFFICE/DocumentServer#1117]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
When converting a DOCX to PDF, the FOOTER of the document with the current page + the total number of pages is corrupted, and the total value of the pages does not appear. This happens for documents that have more than 7 pages, starting on page 7 the others do not appear.

![image](https://user-images.githubusercontent.com/22328594/103371336-f5e2a100-4aad-11eb-8908-8a6b5b748b60.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
To reproduce, just take a DOCX with more than 7 pages and define in the FOOTER of the document the numbering of the current page + total pages, and then convert to PDF. This by the API.

**What is the expected behavior?**
The total number of pages appears normally in the generation, as in previous pages.

**Did this work in previous versions of DocumentServer?**
Not tested.

**DocumentServer version:**
5.5.3

**Operating System:**
Ubuntu 18.04.3 LTS

**Browser version:**
Using the API

**Comment by ShockwaveNN** [comment]
> DocumentServer version:
> 5.5.3

Hi, could you update DocumentServer to latest version, 5.5.3 is pretty old and there is a good chance that we fixed problem since then

**Comment by ShockwaveNN** [comment]
This issue was closed due to no response.

**Comment by cpot** [comment]
@ShockwaveNN We also have this annoying issue. please reopen

**Comment by ShockwaveNN** [comment]
@cpot Could you create a new issue with example docx please? I don't think reopening issue with no response from author is a good idea

**Comment by cpot** [comment]
please find the example attached. i suggest you reopen this one because it seems it is an old issue

[numbering problem with PDF API Onlyoffice.docx](https://github.com/ONLYOFFICE/DocumentServer/files/6732033/numbering.problem.with.PDF.API.Onlyoffice.docx)

![image](https://user-images.githubusercontent.com/3287514/123769857-02fc1380-d8ca-11eb-8943-dfb6bdfa32fd.png)

**Comment by ShockwaveNN** [comment]
@cpot Do you use api to convert this file, or result is the same if I open your docx in web editor and save to PDF?

**Comment by cpot** [comment]
Yes the problem is only with the API (like a lot of other issues), if you Print as PDF or Save, no problem. And of course no problem in the editor

**Comment by ShockwaveNN** [comment]
@cpot got it, thanks. Will take a look soon

**Comment by ShockwaveNN** [comment]
I've created issue 51026 in our private issue tracker

**Comment by ShockwaveNN** [comment]
Hi there, issue 51026 actually is a duplicate of similar issue in our tracker - 50640

And this issue recently fixed - https://github.com/ONLYOFFICE/sdkjs/commit/618c9873c794befc223b32ea32637cd06d37cd39

This should be released as v6.4.0, but currently I have no time frame for this release

If you want to recheck this issue - you can use nightly docker `onlyoffice/4testing-documentserver-ee:6.4.0.79` - but please note that it's still not production ready
